com.github.slgobinath.SafeEyes.yaml

```diff
@@ -0,0 +1,99 @@
+id: com.github.slgobinath.SafeEyes
+sdk: org.freedesktop.Sdk
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+command: safeeyes
+
+finish-args:
+  - "--share=network"
+  - "--share=ipc"
+  - "--socket=x11"
+  - "--socket=pulseaudio"
+  - "--device=dri"
+  - "--system-talk-name=org.freedesktop.DBus" # for tray icon
+  - "--talk-name=org.freedesktop.Notifications" # for 'notification before breaks' plugin
+  - "--talk-name=org.mpris.MediaPlayer2.*" # to be able to pause media player on break
+  # various DE tray support
+  - "--talk-name=com.canonical.indicator.application"
+  - "--talk-name=com.canonical.AppMenu.Registrar"
+  - "--talk-name=org.ayatana.indicator.application"
+  - "--talk-name=org.gnome.Mutter.IdleMonitor"
+  - "--talk-name=org.kde.StatusNotifierWatcher"
+  - "--env=GTK_PATH=/app/lib/gtk-3.0"
+
+cleanup: # Cleanup after the python modules
+  - /cache
+  - /man
+  - /share/aclocal
+  - /share/devhelp
+  - /include
+  - /lib/pkgconfig
+  - /share/gtk-doc
+  - /share/man
+  - /lib/*.a
+  - /lib/*.la
+
+modules:
+  - shared-modules/libappindicator/libappindicator-gtk3-introspection-12.10.json # app indicator support
+  - shared-modules/libappindicator/libappindicator-gtk3-12.10.json
+  - libcanberra/libcanberra.json # don't clean /lib/gtk-3.0
+
+  # auto generated pip dependencies for safeeyes
+  # install pycairo first or build fails
+  # install order is: pycairo -> setuptools-scm -> safeeyes
+  # `python3 flatpak-pip-generator.py  --runtime='org.freedesktop.Sdk//22.08' PKGNAME`
+  - pypi-dependencies.json
+
+  - name: dbus-python # dbus - for tray
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: "https://dbus.freedesktop.org/releases/dbus-python/dbus-python-1.3.2.tar.gz"
+        sha256: ad67819308618b5069537be237f8e68ca1c7fcc95ee4a121fe6845b1418248f8
+
+  - name: wlrctl # required for 'wayland do not disturb' plugin
+    buildsystem: meson
+    sources:
+      - type: archive
+        url: "https://git.sr.ht/~brocellous/wlrctl/archive/v0.2.1.tar.gz"
+        sha256: de8848de0a81749fe7ee7316bead0261bd978e26c2f899d02897609972a6e98c
+
+  - name: alsa-utils # required for 'audible alert' plugin
+    buildsystem: simple
```

Comment:
this should be `autotools` buildsystem.

Reply:
autotools without build-commands right?
Without prefix, build will fail 

```
/usr/bin/install: cannot create regular file '/lib/udev/rules.d/90-alsa-restore.rules': Read-only file system
```

Reply:
`config-opts`